project/plugins.sbt

```diff
@@ -1,11 +1,12 @@
 addSbtCoursier
-addSbtPlugin("org.lyranthe.sbt" % "partial-unification" % "1.1.0")
-//addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.3")
```

Comment:
I don't know why we weren't using it, but seems to be good and we use it in other situations